src/main/java/org/opendevstack/provision/model/rundeck/Job.java

```diff
@@ -16,18 +16,38 @@
 
```

Comment:
Should this be moved to another package? `rundeck.Job` looks weird when there is no Rundeck at all :)